app.js

```diff
@@ -48,23 +47,29 @@ if (!!majorVersion && majorVersion < 14) {
       useCreateIndex: true,
       serverSelectionTimeoutMS: 2500
     })
-    .then(() => ensureSystemSettingsInitiated())
+    .then(async (mg) => {
+      await runMigrations(mg.connection.db, mg.connection.getClient());
```

Comment:
FYI: the migrations are run before anything else. They thus can prevent failures, which I think is good thing. If there are bugs in the migrations this can be very painful for users as they would often have to nuke the database anyway.

If we design our migrations to be highly secure and tested, there should be no issues during this step. Its thus of great importance that we thoroughly test any new migrations on dev/canary before future releases.